core/src/main/java/org/elasticsearch/index/mapper/ip/IpFieldMapper.java

```
@@ -285,6 +287,14 @@ public FieldStats stats(IndexReader reader) throws IOException {
             failIfNoDocValues();
             return new DocValuesIndexFieldData.Builder().numericType(NumericType.LONG);
         }
+
+        @Override
+        public ValueFormat valueFormat(@Nullable String format, DateTimeZone timeZone) {
+            if (format != null) {
+                throw new IllegalArgumentException("Field [" + name() + "] of type [" + typeName() + "] does not support custom formats");
+            }
```

should we also throw an exception if timeZone is specified here?